core/entity/data/v1/v1.go

```diff
@@ -27,6 +27,15 @@ type (
 	SecretFormat string
 )
 
+func (t JsonTime) MarshalJSON() ([]byte, error) {
+	stamp := fmt.Sprintf("\"%s\"", time.Time(t).Format(time.RubyDate))
+	return []byte(stamp), nil
+}
+
+func (t JsonTime) String() string {
+	return time.Time(t).Format(time.RFC3339)
+}
```

Comment:
this fixes JSON time formatting issues when displaying secret meta info on Sentinel.